### PR TITLE
Automatically create GitHub release with APKs

### PR DESCRIPTION
**Pull request by chrisbanes**
This new action will trigger when we create a tag. I'll test this on `1.0.0-alpha03` once merged.

### CODE REVIEW

File: .github/workflows/Release.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          #tag_name: "${{ github.ref }}"
+          #release_name: ${{ github.ref }}
+          draft: true # TODO update
+          release_name: "Test"
+          prerelease: false
```
Comment (JoseAlcerreca) [inline .github/workflows/Release.yml:38]: should this be true until stable? does it do anything apart from showing the label?

Comment (chrisbanes) [reply .github/workflows/Release.yml:38]: I don't use pre-release personally. It doesn't do anything other than display a small note under the release name.

File: .github/workflows/Release.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      - name: Build all projects
+        run: ./scripts/gradlew_recursive.sh assembleDebug
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: "${{ github.ref }}"
```
Comment (JoseAlcerreca) [inline .github/workflows/Release.yml:34]: Since you just uncommented this, can you reenable the build again to see how this looks like (and if it works)

Comment (chrisbanes) [reply .github/workflows/Release.yml:34]: It won't work, the ref will be a commit id here. Only for the tag will the ref be correct.

Comment (chrisbanes) [reply .github/workflows/Release.yml]: ...and this definitely works. It's taken from the official action docs: https://github.com/actions/create-release

**Review by nickbutcher — APPROVED** [review]

**Pull request merged**